composable_kernel/include/config.hpp

```diff
@@ -151,6 +151,11 @@
 #define CK_WORKAROUND_SWDEV_XXXXXX_THREAD_WISE_COPY_V1R5_TYPE_CONVERT_ISSUE 1
 #endif
 
+// workaround for compiler issue when bwd fp16 use some tuning parameter
+#ifndef CK_WORKAROUND_SWDEV_325164
```

Comment:
I don't see this variable being used

Reply:
done

Reply:
I still don't see this macro being used anywhere in the kernel